common/src/main/java/com/google/auto/common/MoreElements.java

```diff
@@ -189,5 +195,24 @@ public static boolean isAnnotationPresent(Element element,
     return Optional.absent();
   }
 
+  /**
+   * Returns a {@link Predicate} that can be used to filter elements by {@link Modifier}.
+   * <p>
+   * Here is an example how one could get a List of static methods of a class:
+   * <code>
+   * FluentIterable.from(ElementFilter.methodsIn(clazzElement.getEnclosedElements()))
+   *     .filter(MoreElements.hasModifiers(Modifier.STATIC).toList();
+   * </code>
+   */
+  public static Predicate<Element> hasModifiers(Modifier... modifiers) {
```

Comment:
I think I would have a preference for a Set<Modifier> here. Callers can easily use ImmutableSet.of or EnumSet.of.